packages/documentation/.storybook/preview-body.html

```diff
@@ -0,0 +1 @@
+<script src="/assets/scripts/storybook-preview-events.js"></script>
```

Comment:
It could have been part of the manager, but it would have required more work on the observer and I think this event could be beneficial as well for e2e tests, for example, where we only load the iframe not the whole storybook.